internal/go-symlink-logfiles-watcher/Docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM openshift/origin-release:golang-1.15 AS build
```

Comment:
Do we want a single image with two entrypoints, or two images? I was assuming two images, since we will deploy this as part of the fluentd DaemonSet not with the CLO.